docs/tutorials/fundamentals/viewer.md

```diff
@@ -340,39 +299,58 @@ viewer.add_image(blobs, name='blobs', opacity=0.5, colormap='red')
 nbscreenshot(viewer, alt_text="A 2d view of the moon on top of which is overlaid a 3d volume containing blobs through which you can navigate using the dimension slider.")
```

Comment:
This screenshot isn't showing in the built docs—I think it could bebecause of the tag being removed, above.

Reply:
We will soon see. Thanks, Peter.

Reply:
I'll ask @melissawm